portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -25,6 +26,7 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
     private List<String> quotes;
+    private ArrayList<String> m;
```

Comment:
Consider using a more verbose variable name, like `messages`. Typically in Java, single-letter messages are used in only boilerplate scenarios such as loop index variables.